### ISSUE TITLE
PHP Fatal error:  Duplicate type null is redundant

### ISSUE THREAD

**Issue by thomasschiet** [mockery/mockery#1204]
Since upgrading to PHP 8.2, I cannot mock `SplFileObject`.

```php
Mockery::mock(SplFileObject::class, ['php://memory']);
```

This yields the following error:

```
PHP Fatal error:  Duplicate type null is redundant in /app/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php(34) : eval()'d code on line 1174

In EvalLoader.php(34) : eval()'d code line 1174:
                                    
  Duplicate type null is redundant  
                                    
```

Happens on versions 1.5.1 and dev-master 3c3ca93.

**Comment by thomasschiet** [comment]
The problem seems to be with the `getChildren` method on `SplFileObject`, which has the `null` standalone type since PHP 8.2.

The generated code is:
```
    public function getChildren(): null|null{
$argc = func_num_args();
$argv = func_get_args();
$ret = $this->_mockery_handleMethodCall(__FUNCTION__, $argv);
return $r
```

**Comment by ghostwriter** [comment]
Resolved via #1205
